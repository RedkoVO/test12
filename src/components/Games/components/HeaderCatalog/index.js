import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import UserImg from '../../../../assets/images/user.png'

import Filters from '../Filters'

import styles from './styles'

const HeaderCatalog = ({ classes, isDesktop }) => (
  <div className={classes.root}>
    <div className={classes.catalogUserBlock}>
      <p className={classes.catalogUserHello}> Hello, Ben A <br />
        <span className={classes.catalogUserVerification}>KYC VERIFICATED</span>
      </p>
      <div className={classes.catalogUserPhoto}>
        <img src={UserImg} alt="" />
      </div>
      <p className={classes.catalogUserBallance}> 764.23 DCB <br />
        <span className={classes.catalogUserBallDescr}>YOUR BALANCE</span>
      </p>
    </div>
    
    {isDesktop && <Filters />}
  </div>
)

HeaderCatalog.propTypes = {
  classes: PropTypes.object,
  isDesktop: PropTypes.bool
}

export default withStyles(styles)(HeaderCatalog)