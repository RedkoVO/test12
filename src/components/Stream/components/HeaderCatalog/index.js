import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import UserImg from '../../../../assets/images/user.png'

import styles from './styles'

const HeaderStream = ({ classes }) => (
  <div className={classes.root}>
    <div className={classes.streamUserBlock}>
      <p className={classes.streamUserHello}> Hello, Ben A <br />
        <span className={classes.streamUserVerification}>KYC VERIFICATED</span>
      </p>
      <div className={classes.streamUserPhoto}>
        <img src={UserImg} alt="" />
      </div>
      <p className={classes.streamUserBallance}> 764.23 DCB <br />
        <span className={classes.streamUserBallDescr}>YOUR BALANCE</span>
      </p>
    </div>
  </div>
)

HeaderStream.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(HeaderStream)