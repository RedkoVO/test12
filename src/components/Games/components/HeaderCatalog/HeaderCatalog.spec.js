import * as React from 'react'
import renderer from 'react-test-renderer'

import HeaderCatalog from './'

it('renders correctly DESKTOP', () => {
  const tree = renderer.create(<HeaderCatalog isDesktop={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders correctly MOBILE', () => {
  const tree = renderer.create(<HeaderCatalog isDesktop={false} />).toJSON()
  expect(tree).toMatchSnapshot()
})