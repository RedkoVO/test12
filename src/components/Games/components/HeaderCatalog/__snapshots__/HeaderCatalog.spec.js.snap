// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly DESKTOP 1`] = `
<div
  className="HeaderCatalog-root-1"
>
  <div
    className="HeaderCatalog-catalogUserBlock-2"
  >
    <p
      className="HeaderCatalog-catalogUserHello-3"
    >
       Hello, Ben A 
      <br />
      <span
        className="HeaderCatalog-catalogUserVerification-5"
      >
        KYC VERIFICATED
      </span>
    </p>
    <div
      className="HeaderCatalog-catalogUserPhoto-7"
    >
      <img
        alt=""
        src="user.png"
      />
    </div>
    <p
      className="HeaderCatalog-catalogUserBallance-4"
    >
       764.23 DCB 
      <br />
      <span
        className="HeaderCatalog-catalogUserBallDescr-6"
      >
        YOUR BALANCE
      </span>
    </p>
  </div>
</div>
`;

exports[`renders correctly MOBILE 1`] = `
<div
  className="HeaderCatalog-root-20"
>
  <div
    className="HeaderCatalog-catalogUserBlock-21"
  >
    <p
      className="HeaderCatalog-catalogUserHello-22"
    >
       Hello, Ben A 
      <br />
      <span
        className="HeaderCatalog-catalogUserVerification-24"
      >
        KYC VERIFICATED
      </span>
    </p>
    <div
      className="HeaderCatalog-catalogUserPhoto-26"
    >
      <img
        alt=""
        src="user.png"
      />
    </div>
    <p
      className="HeaderCatalog-catalogUserBallance-23"
    >
       764.23 DCB 
      <br />
      <span
        className="HeaderCatalog-catalogUserBallDescr-25"
      >
        YOUR BALANCE
      </span>
    </p>
  </div>
</div>
`;
